webapp/components/invite_member_modal.jsx

```diff
@@ -59,6 +59,8 @@ class InviteMemberModal extends React.Component {
         this.removeInviteFields = this.removeInviteFields.bind(this);
         this.showGetTeamInviteLinkModal = this.showGetTeamInviteLinkModal.bind(this);
 
+        const team = TeamStore.getCurrent();
```

Comment:
I'm not sure what you mean? There is already a listener for if the team changes after this step